app/api/currentuser/reject_group_invitation.go

```diff
@@ -6,6 +6,36 @@ import (
 	"github.com/France-ioi/AlgoreaBackend/app/service"
 )
 
+// swagger:operation POST /current-user/group-invitations/{group_id}/reject groups users groupInvitationReject
+// ---
+// summary: Reject a group invitation
+// description:
+//   Let a user reject an invitation to join a group.
+//   On success the service sets `groups_groups.sType` to `invitationRefused` and `sStatusDate` to current UTC time.
+//
+//   * There should be a row in `groups_groups` with the `group_id` as a parent
+//   and the authenticated user’s selfGroup’s `ID` as a child with `sType`=`invitationSent`/`invitationRefused`.
+//   Otherwise the unprocessable entity error is returned.
+//
+//   * If `groups_groups.sType` is `invitationRefused` already, the "unchanged" (200) response is returned.
+// parameters:
+// - name: group_id
+//   in: path
+//   type: integer
+//   required: true
+// responses:
+//   "200":
+//     "$ref": "#/responses/updatedOrNotChangedResponse"
```

Comment:
"Unchanged" instead of "NoChanged" here as well ? 🤔

Reply:
fixed